app/services/teachers/induction_period.rb

```diff
@@ -43,5 +43,9 @@ def overlapping_with?(date)
     teacher.induction_periods.ongoing_on(date).exists?
   end
 
-  delegate :last_induction_period, :first_induction_period, to: :teacher
+  delegate :first_induction_period, to: :teacher
+
+private
+
+  def last_induction_period = teacher.last_induction_period
```

Comment:
This might look kind of weird, but I was running into Rubocop issue trying to delegate a method "privately". (Is that even possible? 🤔)